src/DynamicExpresso.Core/Parsing/Parser.cs

```diff
@@ -3340,5 +3336,73 @@ private static T[] RemoveLast<T>(T[] array)
 				return result;
 			}
 		}
+
+		/// <summary>
+		/// Binds to a member access of an instance as late as possible.  This allows the use of anonymous types on dynamic values.
+		/// </summary>
+		private class LateGetMemberCallSiteBinder : CallSiteBinder
+		{
+			private readonly string m_propertyOrFieldName;
```

Comment:
To be consistent with naming conventions, please use `_propertyOrFieldName`

Reply:
Resolved in latest commit